src/openMVG/calibration/patternDetect.hpp

```diff
@@ -11,10 +12,8 @@ enum Pattern
 {
   CHESSBOARD = 0,
   CIRCLES_GRID,
-  ASYMMETRIC_CIRCLES_GRID
-#ifdef HAVE_CCTAG
-  , CCTAG_GRID
-#endif
+  ASYMMETRIC_CIRCLES_GRID,
+  ASYMMETRIC_CCTAG_GRID
```

Comment:
I'm not sure, if it's a good idea to change the enum values (associated to indexes) if you build with or without cctags. For instance, if you rebuild with an additional dependency, the ID could have changed... Currently there is nothing after, but it may happen.

So we have 2 solutions: use explicit values for enums or keep all enum entries even if it's not builded with and throw inside the "switch/case".
I'm not sure what is the best.